src/opt_mo/evolutionary_best_response.py

```diff
@@ -49,4 +50,11 @@ def get_evolutionary_best_response(
         history.append(best_response)
         cycle, repeat_length = get_repeat_cycle_and_length(history, tol=tol)
 
+    utilities = [
+        opt_mo.tournament_utility(player, opponents)
+        + opt_mo.tournament_utility(player, [player] * K)
```

Comment:
Is the following slightly nicer?

```suggestion
        opt_mo.tournament_utility(player, opponents + [player] * K)
```